aas-gui/Frontend/aas-web-gui/src/components/AppNavigation/AASList.vue

```diff
@@ -44,7 +44,7 @@
                         </template>
                         <!-- idShort of the AAS -->
                         <template v-if="!drawerState" v-slot:title>
-                            <div class="text-primary" style="z-index: 9999">{{ AAS['idShort'] }}</div>
+                            <div v-html="aasNameToDisplay(AAS)" class="text-primary" style="z-index: 9999"></div>
```

Comment:
I would suggest doing this instead to prevent potential HTML injections (not saying that this would happen here but as a general best practice):

`<div class="text-primary" style="z-index: 9999">{{ aasNameToDisplay(AAS) }}</div>`